py/mpprint.h

```diff
@@ -39,6 +39,12 @@
 #define PF_FLAG_ADD_PERCENT       (0x100)
 #define PF_FLAG_SHOW_OCTAL_LETTER (0x200)
 
+#if MICROPY_PY_IO
+#    define MP_PRINT mp_sys_stdout_print
```

Comment:
There is not a single define in the MicroPython repository starting with ``MP_PYTHON_``. The common denominator for defines seems to by that they start with ``MP_``, and often ``MP_<"filename">``. So maybe ``MP_PRINT_PRINTER``, or simply ``MP_PRINTER``?